src/LoyaltyToken.ts

```diff
@@ -12,20 +12,16 @@ export function handleTransfer(event: Transfer): void {
   let loyaltyToken = LoyaltyToken.bind(tokenAddress);
   let fromNewBalance = getTokenBalance(from, loyaltyToken);
   let toNewBalance = getTokenBalance(to, loyaltyToken);
-
   let patronFrom = PatronNew.load(fromString);
   let patronTo = PatronNew.load(toString);
-
   if (patronFrom == null) {
     patronFrom = createDefaultPatron(from, event.block.timestamp);
   }
   patronFrom.currentBalance = fromNewBalance;
-
   if (patronTo == null) {
     patronTo = createDefaultPatron(from, event.block.timestamp);
   }
   patronTo.currentBalance = toNewBalance;
-
   patronFrom.save();
   patronTo.save();
```

Comment:
![Screen Shot 2020-05-09 at 11 02 47](https://user-images.githubusercontent.com/20556729/81469316-a4b33380-91e4-11ea-9cb3-1f326387fa35.png)
This is broken it seems?? Are you pulling the loyalty balance direct from the chain at the moment?

Reply:
No that was fixed in master. But you are working from a stale branch that doesn't include latest master...

I think we should stop deploying to the main wildcards graphs when developing. It is extremely bad actually. Please create a graph under "moose-code" that you use for testing.

I use the value on chain sometimes, and the value here sometimes. The value here doesn't take into account transfers or burns (so thats where I use the onchain value).

Reply:
Cool agree and sorry.

Please check and test it out at https://thegraph.com/explorer/subgraph/moose-code/wilcardsgoerli?query=voteManager%20iterations